jaeger-core/src/main/java/com/uber/jaeger/Configuration.java

```diff
@@ -402,11 +430,7 @@ private static Number getPropertyAsNum(String name) {
   }
 
   private static Boolean getPropertyAsBoolean(String name) {
-    String value = getProperty(name);
-    if (value != null) {
-      return Boolean.valueOf(value);
-    }
-    return null;
+    return Boolean.valueOf(getProperty(name));
```

Comment:
You're changing the semantics of the method, previously it could return null, now it performs defaulting of null string to `false`. It still works given how it's used in this class, but by accident. I would rename it to `getPropertyAsBool` and return `bool`, and add a comment about defaulting.

Reply:
True, I didn't see any cases where it was necessary to differentiate null from false. I'll update as you recommended.